src/Config.cpp

```diff
@@ -12,25 +12,31 @@
 #include "Output.h"
```

Comment:
This file is all whitespace changes and is unrelated to changing the number of multiplexers. Probably shouldn't be included in this PR.

Reply:
Oh wait, I see it changes to include mux header files. Was that intentional?

Reply:
See other comment above.